README.md

```diff
@@ -86,15 +86,15 @@ Airflow is not a streaming solution, but it is often used to process real-time d
 
 Apache Airflow is tested with:
 
-|            | Main version (dev)           | Stable version (2.6.1) |
-|------------|------------------------------|------------------------|
-| Python     | 3.7, 3.8, 3.9, 3.10          | 3.7, 3.8, 3.9, 3.10    |
-| Platform   | AMD64/ARM64(\*)              | AMD64/ARM64(\*)        |
-| Kubernetes | 1.23, 1.24, 1.25, 1.26, 1.27 | 1.23, 1.24, 1.25, 1.26 |
-| PostgreSQL | 11, 12, 13, 14, 15           | 11, 12, 13, 14, 15     |
-| MySQL      | 5.7, 8                       | 5.7, 8                 |
-| SQLite     | 3.15.0+                      | 3.15.0+                |
-| MSSQL      | 2017(\*), 2019(\*)           | 2017(\*), 2019(\*)     |
+|            | Main version (dev)           | Stable version (2.6.1)    |
+|------------|------------------------------|---------------------------|
+| Python     | 3.7, 3.8, 3.9, 3.10, 3.11    | 3.7, 3.8, 3.9, 3.10, 3.11 |
```

Comment:
Are we supporting Python 3.11 in 2.6.1?

Reply:
Nope :). Good catch

Reply:
https://github.com/apache/airflow/pull/31468